pkg/plugins/builtin/callback/handler.go

```diff
@@ -0,0 +1,109 @@
+package plugincallback
+
+import (
+	"github.com/gin-gonic/gin"
+	"github.com/juju/errors"
+	"github.com/loopfz/gadgeto/zesty"
+	"github.com/ovh/utask"
+	"github.com/ovh/utask/engine"
+	"github.com/ovh/utask/models/task"
+	"github.com/ovh/utask/pkg/jsonschema"
+	"github.com/ovh/utask/pkg/metadata"
+	"github.com/sirupsen/logrus"
+)
+
+const (
+	CallbackID     = "callback_id"
+	CallbackSecret = "callback_secret"
+	CallbackBody   = "callback_body"
+)
+
+type handleCallbackIn struct {
+	CallbackID     string                 `path:"id, required"`
+	CallbackSecret string                 `query:"t, required"`
+	Body           map[string]interface{} `body:""`
+}
+
+type handleCallbackOut struct {
+	Message string `json:"message"`
+}
+
+func HandleCallback(c *gin.Context, in *handleCallbackIn) (res *handleCallbackOut, err error) {
+	metadata.AddActionMetadata(c, CallbackID, in.CallbackID)
+	metadata.AddActionMetadata(c, CallbackSecret, in.CallbackSecret)
+	metadata.AddActionMetadata(c, CallbackBody, in.Body)
+
+	dbp, err := zesty.NewDBProvider(utask.DBName)
+	if err != nil {
+		return nil, err
+	}
+
+	if err = dbp.Tx(); err != nil {
+		return nil, err
+	}
+
+	cb, err := loadFromPublicID(dbp, in.CallbackID)
```

Comment:
We should do a loadLockedFromPublicID, that creates a SELECT FOR UPDATE in database.
In that way, it will prevent to have a callback triggered twice at the same time